images/router/haproxy/conf/haproxy-config.template

```diff
@@ -434,6 +435,7 @@ backend openshift_default
   option forwardfor
   #option http-keep-alive
   option http-pretend-keepalive
+  http-request deny deny_status 404
```

Comment:
This doesn't belong here any more, right?

Reply:
I did not get what you mean ? It main thing of this PR :)

Reply:
It is required to trigger 404 as haproxy generates 503.

Reply:
Ok, I misunderstood.